Source/relay/TourGuide/Items of the Month/2019/Kramco Sausage-o-Matic.ash

```diff
@@ -31,7 +31,7 @@ void IOTMKramcoSausageOMaticGenerateTasks(ChecklistEntry [int] task_entries, Che
 RegisterResourceGenerationFunction("IOTMKramcoSausageOMaticGenerateResource");
 void IOTMKramcoSausageOMaticGenerateResource(ChecklistEntry [int] resource_entries)
 {
-    if (!__iotms_usable[lookupItem("Kramco Sausage-o-Matic")] || my_path().id == PATH_LIVE_ASCEND_REPEAT) return;
+    if (!__iotms_usable[lookupItem("Kramco Sausage-o-Matic &trade;")] || my_path().id == PATH_LIVE_ASCEND_REPEAT) return;
```

Comment:
For all the Kramco ones, make sure there's no space between Matic and the trade symbol:
```
> ash $item[Kramco Sausage-o-Matic &trade;]

Bad item value: "Kramco Sausage-o-Matic &trade;" (char 7 to char 37)
Returned: 0

> ash $item[Kramco Sausage-o-Matic&trade;]
Returned: Kramco Sausage-o-Matic™
id => 10058
[...etc]
```

Reply:
addressed in latest push. thank you zinc! <3